### ISSUE TITLE
Experiment status information

### ISSUE THREAD

**Issue by amurphy-cl** [CarnegieLearningWeb/educational-experiment-service#105]
After an experiment status has changed to Enrolling, you can't see when enrollment started. We have when the experiment was "created on", but will need to know when enrollment was started and when it was completed, during and after experiment enrollment.

**Comment by jerith666** [comment]
I think this info is in the audit log, isn't it?

**Comment by amurphy-cl** [comment]
Yes, but we shouldn't need to dig into the audit log to get this info. It's being captured, just needs to be displayed somewhere on the dashboard.

**Comment by nirmalpatel** [comment]
@jerith666 @amurphy-cl we can have something like this:

![image](https://user-images.githubusercontent.com/830400/110105957-0de8e080-7dcf-11eb-9bb5-db620c621ffb.png)

I'm still not sure what to display as the enrollment start time when the experiment is back enrolling after it has been set to enrollment complete once. Maybe we can just overwrite the enrollment start time every time the experiment starts and erase the enrollment end time... or we can store arrays of timestamps and always display the first/last item in the array.

**Comment by amurphy-cl** [comment]
This looks great! Good point about re-enrolling, though. I don't think we'd want to completely overwrite the start time, but storing arrays might make sense (with a button to open up a window with "History", perhaps). Another option could be displaying multiple rows for start/end if there are multiple starts/ends since there aren't expected to be a lot of these, but could clutter the UI.

**Comment by SritterCL** [comment]
Maybe at the top level, we just display the earliest enrollment and, if the experiment is not currently enrolling, the latest enrollment complete time. We could also display "X enrollment periods" (where X is the number of times the experiment started enrolling) and clicking on that gives you the details about the enrollment periods.
